tests/track/patches/test_client.py

```diff
@@ -0,0 +1,136 @@
+from unittest.mock import patch
+
+from track.patches.client import AuthorizedClient, patch_aim_client
+
+
+@patch('track.patches.client.Client.connect')
+def test_authorized_client_adds_bearer_token(mock_connect):
+    """Test that AuthorizedClient adds Authorization header."""
+    # Set a bearer token
+    AuthorizedClient.BEARER_TOKEN = 'test_token_123'
+
+    client = AuthorizedClient('http://test.com')
```

Comment:
`test.com` is a real site. We should use a reserved site like `test.example` (which doesn't resolve) or `example.com` (which resolves but is special).